src/workingtitle-vcockpits-instruments/html_ui/Pages/VCockpit/Instruments/Shared/Map/WorkingTitle/Svg/SvgAltitudeInterceptElement.js

```diff
@@ -105,7 +105,7 @@ class SvgAltitudeInterceptElement extends SvgMapElement {
         return new Vec2(_center.x + _radius * Math.sin(_angle), _center.y - _radius * Math.cos(_angle));
     }
 }
-SvgAltitudeInterceptElement.STROKE_COLOR_DEFAULT = "cyan";
+SvgAltitudeInterceptElement.STROKE_COLOR_DEFAULT = "white";
```

Comment:
Please don't change the defaults in SvgAltitudeInterceptElement. This will affect every instance of the class that is created. If you want to change the colors for your specific implementation, use either mapconfig.json or set the appropriate property on your instance of the class only.

Reply:
Oops!